app/src/models/subscan.ts

```diff
@@ -0,0 +1,150 @@
+import { TransferStatus } from '@snowbridge/api/dist/history'
+
+export type SubscanEvent = {
+  id: number
+  block_timestamp: number
+  event_index: string
+  extrinsic_index: string
+  phase: number
+  module_id: string
+  event_id: string
+  extrinsic_hash: string
+  finalized: boolean
+}
+
+export type SubscanExtrinsicEvent = {
+  event_index: string
+  block_num: number
+  extrinsic_idx: number
+  module_id: string
+  event_id: string
+  params: string
+  phase: number
+  event_idx: number
+  extrinsic_hash: string
+  finalized: boolean
+  block_timestamp: number
+}
+
+export type SubscanParams = {
+  type: string
+  type_name: string
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  value: any
+  name: string
+}
+
+export type ParachainToETHTransferResult = {
```

Comment:
`nit` but it would be nice to rename `ETH` to `Eth` in these names  and abbreviate "Transfer" to `Tx` to make it easier to read.

```suggestion
export type ParachainToEthTxResult = {
```

Another suggestion, that I leave up to you., is to namespace these types so that the name doesn't have to be so extensive.

For example:

``` typescript 

namespace TxResult {
  type ParaToEth {
    ...
  }

  type SubscanXcm {
    ...
   }
}
```

And then we can reference those types a `TxResult.ParaToEth`, `TxResult.SubscanXcm`, etc.

Reply:
This is nice.

_I'm adding this lead to the cleaning part in [this task](https://linear.app/velocitylabs/issue/VEL-65/tracking-transfer-code-base-architecture-cleaning)._